backend/app/models/Schedule.py

```diff
@@ -0,0 +1,19 @@
+import uuid
+
+from sqlalchemy import Column, DateTime, Enum, Interval, Integer, ForeignKey
+from sqlalchemy.orm import relationship
+from app.models.ScheduleState import ScheduleState
+
+from .Base import Base
+
+
+class Schedule(Base):
+    __tablename__ = "schedules"
+
+    id = Column(Integer, primary_key=True)
+    scheduled_time = Column(DateTime, nullable = True)
```

Comment:
Let's rename this to `selected_time` since the word `schedule` is being overloaded in this model.